src/main/java/core/basesyntax/OperationStrategy.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public class OperationStrategy {
+    public void execute(ShopStorage storage, FruitTransaction transaction) {
+    }
+}
```

Comment:
Refactor `OperationStrategy` to properly implement the strategy pattern. It should be an interface or abstract class defining a method to retrieve the appropriate `OperationHandler` based on the transaction type, rather than having an `execute` method.